library/cartridge_check_instance_not_started.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+from ansible.module_utils.helpers import Helpers as helpers
+
+argument_spec = {
+    'console_sock': {'required': True, 'type': 'str'},
+}
+
+
+def check_state(params):
+    if helpers.is_instance_started(params['console_sock']):
+        return helpers.ModuleRes(failed=True, msg='instance is not stopped')
```

Comment:
What if the instance wasn't started? Maybe `msg='Instance is running'` is better?